Lib/test/test_asyncio/utils.py

```diff
@@ -551,8 +551,10 @@ def _get_running_loop():
             frame = sys._getframe(1)
 
             if frame.f_globals['__name__'] == 'asyncio.mixins':
-                # When we called from LoopBoundedMixin we should
-                # fallback to default implementation of get_running_loop
+                # When we called from LoopBoundMixin we should fall back
+                # to the default implementation of get_running_loop because
+                # we must return the currently running event loop
+                # rather than None
```

Comment:
Ah, now I get it. Perhaps we should refactor our test suite to make sure that `get_running_loop()` works correctly under it, @asvetlov

Reply:
I thought about it but from what I saw it will require a lot of work so that why I chose an easier way)

Reply:
@1st1 I was curious about what will be if we will remove patching of `_get_running_loop` and when I removed it all tests were passed🤩

Looks like either I missed smth very important, either this patching is no longer required.

Reply:
@1st1 I have gone ahead and remove `_ger_running_loop`.

Sorry in advance if I violate some rules, I have just thought that if it doesn't break anything then we can remove it.

Reply:
I think it's better without it.